lib/Service/NotesService.php

```diff
@@ -190,7 +190,10 @@ public function update($id, $content, $userId, $category = null, $mtime = 0) {
 		}
 
 		if ($content !== null) {
-			$file->putContent($content);
+			$stream = fopen('php://temp', 'r+');
```

Comment:
You're never closing `$stream` with [fclose](https://php.net/manual/en/function.fclose.php)

Reply:
The stream is [closed by putContent()](https://github.com/nextcloud/server/blob/master/lib/private/Files/View.php#L674)

Reply:
Okay, error-prone ownership management.